doc/api/util.markdown

```diff
@@ -551,6 +551,8 @@ util.isUndefined(null)
 
 ## util.log(string)
 
+    Stability: 0 - Deprecated: Use a third party userland module instead.
```

Comment:
`userland` is maybe redundant here? Maybe also just point to `.debuglog()`? Even though there are no timestamps.